server/s3_object_metadata.h

```diff
@@ -41,7 +41,7 @@
 // to make them at the bottom of object list index table.
 // This special character is chosen, as it is not allowed in S3 object name and
```

Comment:
update the comment

Reply:
ok.

Reply:
will remove #define EXTENDED_METADATA_OBJECT_PREFIX